chart/templates/_config.tpl

```diff
@@ -47,6 +47,9 @@
     {{- $_ := set $ "kubecf" dict }}
   {{- end }}
   {{- if not $.kubecf.manifest }}
+    {{- /* copy all structures under $.Values so our changes do not affect --reuse-values */}}
+    {{- $_ := set $ "Values" (deepCopy $.Values) }}
```

Comment:
```suggestion
    {{- $_ := deepCopy $.Values | set $ "Values" }}
```
No need to take this one — it just felt odd approving something with no comments 😁